include/boost/url/const_string.hpp

```diff
@@ -0,0 +1,216 @@
+//
+// Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_CONST_STRING_HPP
+#define BOOST_URL_CONST_STRING_HPP
+
+#include <boost/url/detail/config.hpp>
+#include <boost/url/string_view.hpp>
+#include <boost/type_traits/make_void.hpp>
```

Comment:
do we still need make_void?

Reply:
I have no idea. I didn't touch that at all.

Reply:
find out